Package@swift-5.8.swift

```diff
@@ -0,0 +1,94 @@
+// swift-tools-version: 5.6
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+import Foundation
+
+let package = Package(
+    name: "Scipio",
+    platforms: [
+        .macOS(.v12),
```

Comment:
How about change it to v13 in  both Package.swift side? Many engineer may use new macOS

Reply:
I'll upgrade the development environment later because I want to remove NSRegularExpression. But I'll do this next time.